src/js/video.js

```diff
@@ -26,6 +26,17 @@ function eventListener(video, ev) {
 	document.body.dispatchEvent(event);
 }
 
+function shouldDispatch(progress) {
+
+	const relevantProgressPoints = [ 8,  9, 10, 11, 12,
+									23, 24, 25, 26, 27,
+									48, 49, 50, 51, 52,
+									73, 74, 75, 76, 77,
+									100];
+
+	return relevantProgressPoints.includes(progress);
```

Comment:
Array.prototype.includes is not in the default set for https://polyfill.io, it will need to be added to the browser features section of orgami.json -- https://github.com/Financial-Times/o-video/blob/master/origami.json#L9